src/js/reducers/feed.js

```diff
@@ -38,18 +38,18 @@ const data = [
     firstname: 'Marah',
     surname: 'Barden',
     photo_url: 'https://scontent.xx.fbcdn.net/v/t1.0-1/p320x320/12279210_10156252155200506_7792104208279318965_n.jpg?oh=f52c8b1712fd11aea00ce928bf1a0dc6&oe=589881E8',
-    _what: [
+    what: [
       'Go to the beach',
       'Go to France',
       'Go Ape'
     ],
-    _where: [
+    where: [
```

Comment:
@minaorangina we don't have anymore this sort of structure of where. We have now where = ['address1', 'address2']